tools/constants.mjs

```diff
@@ -12,7 +12,7 @@ export const STUB_ANT_REGISTRY_ID = 'ant-registry-'.padEnd(43, '1');
 /* ao READ-ONLY Env Variables */
 export const AO_LOADER_HANDLER_ENV = {
   Process: {
-    Id: STUB_PROCESS_ID,
+    Id: STUB_ADDRESS,
```

Comment:
Why this change?

Reply:
This was required to satisfy the authorities checks from AOS on incoming messages